config/settings.yml

```diff
@@ -1109,29 +1109,6 @@ check_in:
     key_path: fake_api_path
     redis_session_prefix: check_in
     timeout: 30
-  chip_api_v1:
```

Comment:
Wondering removing these keys will cause any build issue since we have the keys defined in [devops repository](https://github.com/department-of-veterans-affairs/devops/blob/master/ansible/deployment/config/vets-api/prod-settings.local.yml.j2#L709-L731).  Please check if it is safe to remove here and remove it devops repository as a fast followup

Reply:
It shouldn't be an issue since those keys aren't being used in code anywhere, but yes they can be removed from devops repo in a separate PR

Reply:
+1 to this issue. I don't _think_ there should be an issue since the keys aren't referenced in the code anywhere, but I would remove from the devops repo to be safe

Reply:
Added a PR in devops repo: https://github.com/department-of-veterans-affairs/devops/pull/10236